Lib/ldap/ldapobject.py

```diff
@@ -76,7 +76,9 @@ class NO_UNIQUE_ENTRY(ldap.NO_SUCH_OBJECT):
 
 class SimpleLDAPObject:
   """
-  Drop-in wrapper class around _ldap.LDAPObject
+    This basic class wraps all methods of the underlying C API object.
```

Comment:
Please also indent the leading and trailing ``"""``.

Reply:
That would mean re-indenting the entire class, which is too much for this PR. So I un-indented the contents instead.